spring-cloud-stream/src/main/java/org/springframework/cloud/stream/config/ChannelBindingServiceProperties.java

```diff
@@ -33,16 +34,10 @@
  */
 @ConfigurationProperties("spring.cloud.stream")
 @JsonInclude(Include.NON_DEFAULT)
-public class ChannelBindingProperties {
-
-	public static final String DESTINATION = "destination";
-
-	public static final String PARTITION_COUNT = "partitionCount";
-
-	public static final String PARTITIONED = "partitioned";
+public class ChannelBindingServiceProperties {
```

Comment:
When ChannelBinderServices gets merged with Binder, sure. The difficulty to naming it as such just now is that each binder has their own configuration so we must cull the classes here a little, I think.